docs/Changelog.rst

```diff
@@ -12,6 +12,9 @@ Changelog
 03-11-2021 ``v2``
     Added a new GET webhooks endpoint to retrieve webhook information :ref:`Retrieving your webhooks<retrieving_your_webhooks>`
 
+15-04-2021 ``v2``
```

Comment:
@kalaisubramaniyan - As per ticket details date is 15-04-2021. So it has to be in sequence in changelog page that's the reason it's not on top. Because we have already released it. It's just updating doc.
https://drukwerkdeal.atlassian.net/jira/people/557058:ddc2656a-d638-4311-b59a-de41cba6138e/boards/234?modal=detail&selectedIssue=API-1052